tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/matter/createMatter.controller.js

```diff
@@ -2175,7 +2175,7 @@
 
                 oMatterProvisionFlags = {
                     "MatterLandingFlag": cm.bMatterLandingPage,
-                    "SendEmailFlag": true
+                    "SendEmailFlag": cm.includeEmail

```

Comment:
SendEmailFlag has been set to true always which is wrong and set that value should be based on check box selection